content/tutorial/tutorial.md

```diff
@@ -12,132 +12,130 @@ redirect_from:
 
-## Setup for the Tutorial {#setup-for-the-tutorial}
+## Mempersiapkan Tutorial {#setup-for-the-tutorial}
 
-There are two ways to complete this tutorial: you can either write the code in your browser, or you can set up a local development environment on your computer.
+Ada dua cara untuk menyelesaikan tutorial ini: Anda dapat menulis kode pada *browser* Anda atau mempersiapan *local development environment* pada komputer Anda.
 
-### Setup Option 1: Write Code in the Browser {#setup-option-1-write-code-in-the-browser}
+### Persiapan Cara 1: Menulis Kode pada Browser {#setup-option-1-write-code-in-the-browser}
 
-This is the quickest way to get started!
+Ini adalah cara tercepat untuk memulai!
 
-First, open this **[Starter Code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)** in a new tab. The new tab should display an empty tic-tac-toe game board and React code. We will be editing the React code in this tutorial.
+Pertama, buka **[kode permulaan](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)** ini pada *tab* baru. *Tab* baru tersebut seharusnya menunjukan *board* permainan *tic-tac-toe* yang kosong dan kode React. Kita akan mengedit kode React tersebut pada tutorial ini.
 
-You can now skip the second setup option, and go to the [Overview](#overview) section to get an overview of React.
+Anda dapat melewati cara kedua untuk persiapan dan menuju ke bagian [Ikhtisar](#overview) untuk mendapatkan ikhtisar dari React.
 
-### Setup Option 2: Local Development Environment {#setup-option-2-local-development-environment}
+### Persiapan Cara 2: Local Development Environment {#setup-option-2-local-development-environment}
 
-This is completely optional and not required for this tutorial!
+Cara ini murni opsional dan tidak wajib pada tutorial ini!
 
 <br>
 
 <details>
 
-<summary><b>Optional: Instructions for following along locally using your preferred text editor</b></summary>
+<summary><b>Opsional: Instruksi untuk mengikuti secara lokal menggunakan editor teks pilihan Anda</b></summary>
 
-This setup requires more work but allows you to complete the tutorial using an editor of your choice. Here are the steps to follow:
+Persiapan ini membutuhkan lebih banyak langkah tetapi membuat Anda dapat menyelesaikan tutorial ini menggunakan editor text pilihan Anda. Berikut adalah beberapa langkah yang harus diikuti:
```

Comment:
`editor text` -> `teks editor`

`pilihan Anda` -> `pilihan anda`

Reply:
I think the translation of `text editor` should be `editor teks` instead of `teks editor`. Any opinion for this?